File: wrappers/mvc/demos/Kendo.Mvc.Examples/Models/Map/Marker.cs
using System;

namespace Kendo.Mvc.Examples.Models
{
    public class Marker
    {
        public Marker(double latitude, double longitude, string name)
        {
            LatLng = new double[] { latitude, longitude };
            Name = name;
        }

        public double[] LatLng { get; set; }
        public string Name { get; set; }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/models/Marker.java
package com.kendoui.spring.models;


public class Marker {
    private double[] LatLng;
    private String Name;
    
    public Marker(double latitude, double longitude, String name)
    {
        LatLng = new double[] { latitude, longitude };
        Name = name;
    }

    public double[] getLatLng() {
        return LatLng;
    }

    public void setLatLng(double[] latLng) {
        this.LatLng = latLng;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        this.Name = name;
    }
}
